### ISSUE TITLE
Make cancellation email routing logic match "ready to register" notification

### ISSUE THREAD

**Issue by jlbooker** [AppStateESS/InternshipInventory#326]
When an internship is cancelled, currently the notification email always goes to the Registrar's Office email address.

We should instead apply the same logic as used in the "Ready to Register" email (`email/ReadyToRegisterEmail.php`), where:

- graduate level internships go to the Graduate School address
- Distance ed internships go to the distance ed address
- International internships go to the international registration address

Basically, make the cancellation email's "to" address/routing logic match the "ready to register" email's addressing logic.

**Comment by jlbooker** [comment]
See existing logic at:
https://github.com/AppStateESS/InternshipInventory/blob/master/class/Email/ReadyToRegisterEmail.php#L140

Cancellation Emails:
https://github.com/AppStateESS/InternshipInventory/blob/master/class/Email/CancelInternshipNotice.php
https://github.com/AppStateESS/InternshipInventory/blob/master/class/Email/IntlInternshipCancelNotice.php

Transition that does the sending:
https://github.com/AppStateESS/InternshipInventory/blob/master/class/WorkflowTransition/CancelTransition.php
